paasta_tools/paasta_api.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
```

Comment:
Can you start a new trend and go ahead and make a folder called "api" to help contain stuff?